WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift

```diff
@@ -517,6 +517,8 @@ extension GutenbergViewController: GutenbergBridgeDelegate {
             DDLogWarn(message)
         case .error:
             DDLogError(message)
+        case .fatal:
+            break
```

Comment:
I'm not sure why the build was failing for me here. I was seeing an error that said the switch statement must be exhaustive.

Reply:
I did an update to the bridge without merging the change to WPiOS 😞 
Since there are not `DDLogFatal()`, I'd rather mix the `.error` and `.fatal` in the same case.

Something like:
` case .error, .fatal:`

Reply:
I've updated this now.